test/js/JsObserverSpec.js

```diff
@@ -0,0 +1,96 @@
+var chai = require('chai'),
+    expect = chai.expect,
+    TestObserver = require("../fixture/TestObserver").TestObserver,
+    PService = require("../fixture/TestObserver").PService,
+    serviceStatus = require("../fixture/TestObserver").serviceStatus,
+    TestObserverOne = require("../fixture/TestObserverOne").TestObserverOne,
+    StateType = require("../../dist/src/ServiceStatus").StateType,
+    SinonStub = require("sinon"),
```

Comment:
shouldn't this be require("sinon").SinonStub   ?

Reply:
this is not used i've removed it. and the other one.